### DISCUSSION TITLE
Vless/vmess/trojan/etc support

### DISCUSSION THREAD

**Discussion by arsen-z** [partout-io/passepartout#591] [Ideas]
Hello!

You guys are doing a great job, I've been using your app (pro version) for quite a while now

Are you planning to add vless/vmess/trojan/etc protocols support? There seems to be no app as nice and neat as Passepartout, so I'm curious.

**Comment by keeshux** [comment]
Thanks for opening this thread. Do you know any Swift/ObjC libraries providing integration with such protocols?

**Reply by antonmedvedev1999** [reply]
Hi!
I also use your app in the Pro tariff and it's cool, especially Apple TV support!
There is an app called Hiddify, which is also available on iOS
https://github.com/hiddify
There is also Amnezia
https://github.com/amnezia-vpn
Perhaps it will be possible to get some information from them?

**Comment by keeshux** [comment]
I'm merging this short discussion into https://github.com/partout-io/partout/issues/248 for my sanity.
